src/App.js

```diff
@@ -1,25 +1,11 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import ArmyBuildPane from './components/ArmyBuildPane'
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <ArmyBuildPane/>
   );
```

Comment:
You can make this a functional component + implicit return 🎉 
```suggestion
  const App = () => <ArmyBuildPane />;
```